client/pages/profile.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import jwtDecode from 'jwt-decode';
+
+export default class Profile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      workouts: [],
+      user: {}
+    };
+  }
+
+  componentDidMount() {
+    const token = window.localStorage.getItem('react-context-jwt');
+    const decoded = jwtDecode(token);
+    this.setState({ user: decoded });
```

Comment:
Usually we let App pass us user related info (as a prop or via context) so we don't have all of our components randomly digging in local storage when they mount. It's ok to grab the token, but i don't think setting state for the user makes much sense here